src/platform/site-wide/tests/sitemap/sitemap-helpers.js

```diff
@@ -56,18 +56,17 @@ function sitemapURLs() {
 
 function runTests(client, segment, only508List) {
   segment.forEach(url => {
+    console.time(url); // eslint-disable-line no-console
     const only508 = only508List.filter(path => url.endsWith(path)).length > 0;
     client
-      .perform(() => {
-        // eslint-disable-next-line no-console
-        console.log(url);
-      })
+      .perform(() => {})
```

Comment:
I imagine this line can be deleted, but I didn't feel like restarting CI to find out.